.scalafix.conf

```diff
@@ -0,0 +1,7 @@
+rules = 
+[ ExplicitResultTypes
+, ProcedureSyntax
+, NoValInForComprehension
+]
+RemoveUnused.imports = true
```

Comment:
im wary of this
ive used intelli-j optimize imports a fair bit and it (not often but it does happen) sometimes removes imports that it thinkgs arent being used but actually are (in particular implicits)

Reply:
I'm not sure that IntelliJ uses their own typechecker, but scalafix uses SemanticDB which in turn uses the Scala type checker, so the likelihood of something going wrong once in the same compiler phase, but not another is hopefully low.

Reply:
I have never encountered this problem with IntelliJ and Lightbend stack in the last 2-3 years. It used to happen from time to time in the earlier days, but not recently. Hope SemanticDB is as mature as IntelliJ at this point. Not sure how compatible/bug-free it will be with Typelevel stack libraries ;)